Makefile

```diff
@@ -45,15 +45,15 @@ connscrape:
 unittest:
 	make -C collector unittest
 
-image: collector unittest
```

Comment:
There is a unit-test job in CI, but the unit-tests are never run with the build options of the production binary. It happened in the past that binaries built for debug or running inside a special environment are not detecting issues.

Reply:
Good call, I'll add a separate job that runs with `Release` the release build option.